pkg/apis/mkscontroller/v1alpha1/mkstask_types.go

```diff
@@ -0,0 +1,30 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genreconciler
```

Comment:
```suggestion
// +genreconciler
```
Please remove this annotation